submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/submission/SubmissionService.kt

```diff
@@ -35,7 +37,7 @@ class SubmissionService(
 
         return waitUntil(
             ofMinutes(SYNC_SUBMIT_TIMEOUT),
-            conditionEvaluator = { queryService.existByAccNoAndVersion(accNo, version) },
+            conditionEvaluator = { requestQueryService.isRequestCompleted(accNo, version) },
```

Comment:
this was hidding bugs as submission was faling during post processing in some test but as this await only check persistence fail those errors  were simple ignored